README.md

```diff
@@ -1,2 +1,105 @@
 # unified-user-data-format
+
 Specification for the unified user data format across the various "Tube" apps (NewPipe, FreeTube, Invidious, etc..)
+
+## Intro
+
+* This specification is a very early draft, and prone to regular changes
+* Unless otherwise specified, all fields are assumed to be `Strings`
+
+
+## Top level structure
+
+#### `"version"`
+
+The [semantic version](https://semver.org/) of the specification.
+
+Conditions for bumping major/minor/patch version: TBD
+Potential suffixes: `-draft`, TBD
+
+#### `"subscriptions"`
+
+An array of Subscription objects. See below for details.
+
+#### `"playlists"`
+
+An array of Playlist objects. See below for details.
+
+### Overview
+
+```javascript
+{
+	"version": "X.Y.Z[-draft/-rc]",
+	"subscriptions": Subscriptions[],
+	"playlists": Playlist[],
+}
+```
+
+## Subscription object
```

Comment:
Could we also include an `id` field too?

For YouTube, it would be the Channel ID.

Reply:
See my previous comment regarding URNs